arch/arm/boot/dts/overlays/digidac1-wm8742-audio-overlay.dts

```diff
@@ -0,0 +1,45 @@
+// Definitions for RRA DigiDAC1 Audio card
+/dts-v1/;
+/plugin/;
+
+/ {
+	compatible = "brcm,bcm2708";
+
+	fragment@0 {
```

Comment:
Create the soundcard in the final fragment, otherwise it may crash when loaded and unloaded at runtime.